lib/laa_crime_forms_common/pricing/nsm/rate_sets.yml

```diff
@@ -0,0 +1,36 @@
+- from: null
+  until: "2022-09-29"
+  values:
+    work_items:
+      preparation: 45.35
```

Comment:
Been wondering what the most appropriate way would be for us to encorporate the new youth court fees - it's a fixed fee that can only be claimed from a certain date - however instead of being added from either Rep Order Date or CNTP date - it can only be added if it's no a Breach of Injunction (and therefore only if the date is the Rep Order Date). To me it still makes sense to add the rate here and have the logic of whether the fee is added into the calculator.

Reply:
*I guess we can directly use the claim_type instead of using the dates to figure out whether the fee is applicable

Reply:
Yes, I think it might be like vat. The yaml file says "for this date range, the vat rate is 0.2. The ruby logic says "if VAT is applicable (i.e. if the firm is vat registered or this is a vat-applicable disbursement), use the rate in the yaml file". Similarly, the yaml file would say "for the December 2024 date range onwards, the youth court fee is ~£600". The ruby logic would say "If the youth court fee is applicable (i.e. if it's not BOI and it's youth court etc) use the rate in the yaml file)".